src/highlight.rs

```diff
@@ -44,6 +45,10 @@ pub static DATA: LazyLock<Data> = LazyLock::new(|| {
     }
 });
 
+static LINK_REGEX: LazyLock<Regex> = LazyLock::new(|| {
+    Regex::new("https?://[^ ]+").unwrap()
```

Comment:
That also captures input that are not actually URLs. For example, `<img src="https://foo.com/bar.jpg" />` will also capture the trailing `"`.

Reply:
Fixed for " specifically.